pipeline/pipeline.py

```diff
@@ -0,0 +1,60 @@
+"""
+Pipeline script that connects the ETL scripts,
+Runs the pipeline in a loop every 1 min.
+"""
+
+import time
+from os import environ
+
+from dotenv import load_dotenv
+
+from extract import fetch_all_plant_data, convert_plant_data_to_csv
+from transform import standardize_country_name, remove_rows_with_null, check_soil_moisture_valid, check_soil_temp_valid, normalize_datetimes, change_temp_and_moisture_to_two_dp
+from load import create_database_connection, insert_into_recordings_table
+
+if __name__ == "__main__":
+
+    load_dotenv()
+
+    while True:
```

Comment:
Do you think it's worth putting this chunk of code in a function, then just calling the function in the if __name__ == "__main__" block? Either way, it's clear and makes sense , but putting it in a function might be slightly better stylistically?